Add duration maximums to tcld generation of certificates (#2224)
temporalio/documentation: docs-src/cloud-context/certificates-issue.md

If you don't have an existing certificate management infrastructure, issue the CA and end-entity certificates using `tcld` or open source tools such as certstrap.

:::info

> The maximum number of CA certificates in a certificate bundle is 16.
> The maximum payload size of a certificate bundle (before base64 encoding) is 32 KB.

Reviewer: ```suggestion
A certificate bundle can contain up to 16 CA certificates.
A certificate bundle can have a maximum payload size of 32 KB before base64 encoding.
```
quick switch to a more active voice - wdyt?

:::

#### Use tcld to generate certificates

CA and end-entity certificates can be generated with `tcld`.

> The maximum duration for a CA certificate is 1 year (`-d 1y`).
> An end-entity certificate must expire before its root CA certificate, so specify its duration appropriately.

Reviewer: ```suggestion
A CA certificate can last up to 1 year (`-d 1y`).
You must set an end-entity certificate to expire before its root CA certificate, so specify its duration appropriately.
```

If you don't have an existing certificate management infrastructure, issue the CA and end-entity certificates using `tcld` or open source tools such as certstrap.

:::info

The maximum number of CA certificates in a certificate bundle is 16.
The maximum payload size of a certificate bundle (before base64 encoding) is 32 KB.

:::

#### Use tcld to generate certificates

CA and end-entity certificates can be generated with `tcld`.

The maximum duration for a CA certificate is 1 year (`-d 1y`).

> 

Reviewer 2: ```suggestion
```
